python/pyfunc-server/pyfuncserver/config.py

```diff
@@ -68,6 +101,33 @@ def __init__(self, model_dir: str):
         self.push_gateway = PushGateway(push_enabled,
                                         push_url,
                                         push_interval)
+        
+        # Publisher
+        sampling_ratio = float(os.getenv(*PUBLISHER_SAMPLING_RATIO))
+        publisher_enabled = str_to_bool(os.getenv(*PUBLISHER_ENABLED))
```

Comment:
Should this be the first check and then we parse / initialise all other fields based on whether this is enabled? For example, if publisher is disabled, we don't have to even bother reading the other config variables, `self.publisher` can be `None`, etc. Wdyt?

Reply:
yeah we can do that, I'll update that